README.md

```diff
@@ -58,7 +58,9 @@ to your [`package.json`](https://docs.npmjs.com/files/package.json#dependencies)
 2. Include and start the profiler at the beginning of your application:
 
     ```js
-    var profiler = require('@google-cloud/profiler').start();
+    var profiler = require('@google-cloud/profiler').start().catch((err) => {
```

Comment:
This problem existed before your change, but it gets more confusing after you change: AFAICT, `start` returns `Promise<void>` so assigning it to a variable named `profiler` doesn't make too much sense to me. I would suggest removing the `var profiler = ` from this statement.

Reply:
Thanks! Done.